objectstore-server/src/config.rs

```diff
@@ -91,6 +91,10 @@ pub struct Sentry {
     pub server_name: Option<Cow<'static, str>>,
     pub sample_rate: f32,
     pub traces_sample_rate: f32,
+    /// If true, sample all traces generated from incoming HTTP requests.
+    /// If false, then the SDK will mirror the sampling decision contained in the `sampled` flag of the incoming trace,
+    /// which in practice could be really low.
+    pub propagate_traces: bool,
```

Comment:
Minor: Intuitively, the option name sounds inverse to me compared to what it does. What if we flip this around and call it something like `inherit_sampling_decision`? The current default of inheriting/honoring is good.